src/utils/RegHubUtil.js

```diff
@@ -47,8 +48,11 @@ module.exports = {
       let repos = _.map(data.results, result => {
         return this.normalize(result);
       });
+      let pageLimit = data.num_pages;
+      // Error in API, more items due to array count
+      let maxResults = data.num_results - 1;
       if (response.statusCode === 200) {
-        repositoryServerActions.resultsUpdated({repos});
+        repositoryServerActions.resultsUpdated({repos, pageLimit, maxResults});
       }
```

Comment:
Often `error` is `null`, but `responseCode` may be 500.

Maybe add an `else` statement that calls `repositoryServerActions.error({error:data.detail});`?